src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,10 @@
+import { useEffect, useState, useContext } from "react";
```

Comment:
Import things as you need them, don't just copy and paste all imports from one component to another. 

You've added Route, routes and link here. You will not use at least two of those. Pasting in this way tells me that either you're unclear what this component is going to do, or you're unclear on what these imports do but because they work somewhere else they are needed.